lib/id3c/cli/redcap.py

```diff
@@ -369,6 +369,50 @@ def metadata(self) -> List[Dict[str, str]]:
         return self._fetch('metadata', {})
 
 
+    def update_metadata(self, metadata: Dict[str, str]) -> int:
+        """
+        Update existing *metadata* in this REDCap project.
+
+        *metadata* must be an iterable of :py:class:``dict``s mapping REDCap
+        field names, form names, and other instrument metadata to values.  The
+        instrument field name, form name, and field type, and field label, at a
+        minimum, must be included.   All keys and values must be strings.
+
+        Any value provided for a field, including the empty string, will
+        overwrite any existing value. From the REDCap API Documentation:
+
+        > Because of this method's destructive nature, it is only available for
+        > use for projects in Development status.
+
+        Returns a count of the number of records updated, as reported by
+        REDCap.
+        """
+        parameters = {
+            'data': json.dumps(metadata),
```

Comment:
This should use `as_json()` from our `id3c.json` module to avoid the same pitfalls that 8f3eaf2953c43c09c0043d96218234b6d177cb84 corrected.

Reply:
Ah, thank you!